CONTRIBUTING

```diff
@@ -7,7 +7,7 @@ agreement. http://www.mpich.org/documentation/contributor-docs/
 standards. https://wiki.mpich.org/mpich/index.php/Coding_Standards
 
 3. Use the git pre-commit hook included with MPICH
-(https://github.com/pmodels/mpich/blob/master/maint/hooks/pre-commit)
+(mpich_root/maint/hooks/pre-commit)
```

Comment:
I understand this change, but if you guys are planning to change the branch name to `main`, I’d like to see the link put back.